src/main/java/com/openlattice/edm/EdmApi.java

```diff
@@ -367,6 +373,13 @@ Void forceRemovePropertyTypeFromEntityType(
     @GET( ENTITY_SETS_BASE_PATH )
     Iterable<EntitySet> getEntitySets();
 
+    @GET( SUMMARY_BASE_PATH )
+    Map<UUID, Stream<PropertySummary>> getAllPropertySummaries( @Body EdmRequest request );
+
+
+    @GET( SUMMARY_BASE_PATH + ID_PATH )
+    Stream<PropertySummary> getPropertySummary( @Path( ID ) UUID propertyTypeId );
```

Comment:
Jackson can't serialize streams as they are intended to be single use. Use Iterable<PropertySummary> instead.